library/Zend/Form/DisplayGroup.php

```diff
@@ -29,6 +29,13 @@
  */
 class Zend_Form_DisplayGroup implements Iterator,Countable
 {
+    /**
+     * Element Id
+     * 
```

Comment:
remove trailing space

Reply:
trailing space is still there

Reply:
I fixed here https://github.com/hungtrinh/zf1-future/commit/28b4a91a7b3a1e6ad5db31e7b3019af414cc1dc0